service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -92,6 +100,26 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             .build();
     }
 
+    @PostMapping("/submitted")
+    public void handleSubmittedEvent(
+        @RequestBody @NotNull CallbackRequest callbackRequest) {
```

Comment:
can `CallbackRequest` be null here?

Reply:
Good spot, removing, can also be formatted better.